G1ANT.Addon.IExplorer/G1ANT.Addon.IE.Tests/Commands/ie.fireevent/ShouldChangeTitle_WhenFireEventOnClick.G1ANT

```diff
@@ -0,0 +1,11 @@
+﻿addon iexplorer version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+ie.open ‴google.pl‴
+ie.fireevent eventname ‴onclick‴ search ‴a.gb_P[href='https://mail.google.com/mail/?tab=wm']‴ by query
```

Comment:
`a.gb_P` - what is it? Isn't that too specific? I mean that a slight modification of google interface might lead this code to fail.

Reply:
Did you address this remark (that was AFAIR `a[href*='mail.google']`)?